datalake/src/main/java/com/sequenceiq/datalake/flow/dr/restore/handler/DatalakeRestoreAwaitServicesStoppedHandler.java

```diff
@@ -0,0 +1,100 @@
+package com.sequenceiq.datalake.flow.dr.restore.handler;
+
+import static com.sequenceiq.datalake.flow.dr.restore.DatalakeRestoreEvent.DATALAKE_RESTORE_SERVICES_STOPPED_EVENT;
+
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.dyngr.exception.PollerException;
+import com.dyngr.exception.PollerStoppedException;
+import com.dyngr.exception.UserBreakException;
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.cloudbreak.eventbus.Event;
+import com.sequenceiq.cloudbreak.exception.CloudbreakApiException;
+import com.sequenceiq.datalake.entity.operation.SdxOperation;
+import com.sequenceiq.datalake.entity.operation.SdxOperationType;
+import com.sequenceiq.datalake.flow.dr.restore.event.DatalakeDatabaseRestoreStartEvent;
+import com.sequenceiq.datalake.flow.dr.restore.event.DatalakeRestoreAwaitServicesStoppedRequest;
+import com.sequenceiq.datalake.flow.dr.restore.event.DatalakeRestoreFailedEvent;
+import com.sequenceiq.datalake.repository.SdxOperationRepository;
+import com.sequenceiq.datalake.service.sdx.PollingConfig;
+import com.sequenceiq.datalake.service.sdx.dr.SdxBackupRestoreService;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.flow.reactor.api.handler.ExceptionCatcherEventHandler;
+import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
+
+@Component
+public class DatalakeRestoreAwaitServicesStoppedHandler extends ExceptionCatcherEventHandler<DatalakeRestoreAwaitServicesStoppedRequest> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DatalakeRestoreAwaitServicesStoppedHandler.class);
+
+    @Value("${sdx.stack.restore.status.sleeptime_sec:20}")
```

Comment:
We don't expect the wait to be longer. How about having a shorted sleep time?

Reply:
I will change this here and in backup.

Reply:
I have decreased these to 4 seconds. Let me know if that seems reasonable.